src/debugger/channel/AdapterChannel.js

```diff
@@ -48,20 +48,28 @@ export class AdapterChannel {
     invariant(!isNaN(requestID));
     let prefix = parts[1];
```

Comment:
Do not call it "prefix", call it "messageName". "prefix" is an implementation details, "messageName" indicates its real meaning.